### PR TITLE
Add EIP-5806: Account abstraction through delegate transaction

### PR DESCRIPTION
**Pull request by Amxx**

### CODE REVIEW

File: EIPS/eip-5806.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+eip: 5806
+title: Delegate transaction
+description: This EIP adds a new transaction type that allows a EOAs to execute arbitrary code through delegation
```
Comment (lightclient) [inline EIPS/eip-5806.md:4]:
```suggestion
description: Adds a new transaction type that allows a EOAs to execute arbitrary code through delegation
```

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-5806.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+author: Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/eip-5806-delegate-transaction/11409
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-10-20
+requires: 7, 155, 1559, 2718, 2929, 2930
```
[the remark this answers is not in the document]
Comment (Amxx) [reply EIPS/eip-5806.md:11]: Some are for reference

I think it needs 2718 to define transactions envelop and 2930 for access list. The rest my be superfluous

**Review by lightclient — APPROVED** [review]
LGTM

**Review by za8457 — COMMENTED** [review]
Amxx:EOA_delegate_transaction

**Review by jskelly84 — APPROVED** [review]

**Review by jskelly84 — APPROVED** [review]

**Pull request merged**